### ISSUE TITLE
Missing path (xxx/Carthage/Build/Lottie.xcframework/ios-arm64_i386_x86_64-simulator/dSYMs) from XCFramework 'Lottie.xcframework' as defined by 'DebugSymbolsPath' in its `Info.plist` file

### ISSUE THREAD

**Issue by Bruce9487** [airbnb/lottie-ios#1457]
## Check these before submitting:
- [v] This issue isn't related to another open issue

## This issue is a:
- [v] Regression (Something that once worked, but doesn't work anymore)

## Which Version of Lottie are you using?

Checking out lottie-ios at "f237d87961d78c08baefd2e134b72fd07eb5c98b"

## What Platform are you on? 

- [v] iOS (XCode 12.2.1)

## What Language are you in?
- [v] Swift

## Actual Behavior
![截圖 2021-12-22 19 02 05](https://user-images.githubusercontent.com/19691907/147083050-dbc9147c-40f0-412f-9bbf-099f25ad0af2.png)

**Comment by calda** [comment]
@Bruce9487, could you share a sample project that reproduces this issue? That would be super helpful.

**Comment by teawithfruit** [comment]
I have the same problem.
It's reproducible with just the normale way you integrate lottie with ```carthage update lottie-ios --platform iOS --use-xcframeworks```.

**Comment by calda** [comment]
I'm unable to reproduce this issue. I wonder if this is a Carthage issue -- what version of Carthage are you using? (You can check by running `$ carthage version`). I'm using Carthage 0.38.0 (the latest version), and it works as expected.

If this isn't related to the Carthage version, then it must be related to the specific build configuration. What type of machine are you using (Intel / Apple Silicon?), and what platform are you compiling for?

**Comment by calda** [comment]
Could you share a sample project that reproduces this issue? That would be very helpful

**Comment by bryanmontz** [comment]
I'm having the exact same issue on an Apple Silicon MBP. Xcode 13.2.1 and Carthage 0.38.0. Happens with a few other libraries as well. Any resolution?

**Comment by calda** [comment]
> Happens with a few other libraries as well.

What other libraries does this happen with? If this is happening with libraries other than Lottie, then it may be an issue with Carthage itself.

**Comment by bryanmontz** [comment]
Yeah I think the issue is with carthage build settings, not Lottie. I'd rather not list the other libraries. I was able to get it to work by running Xcode in Rosetta mode. No other changes.
